ci-operator/step-registry/install-operators/install-operators-commands.sh

```
@@ -78,7 +78,13 @@ for operator_obj in "${OPERATOR_ARRAY[@]}"; do
             -l catalog=${operator_source} \
             -ojsonpath='{.items[?(.metadata.name=="'${operator_name}'")].status.defaultChannel}' 2>/dev/null || echo)
         if [[ -z "${operator_channel}" ]]; then
-            echo "ERROR: Default channel not found in '${operator_name}' packagemanifest."
+            echo "WARNING: Default channel not found in '${operator_name}' packagemanifest. Falling back to the first available channel."
+            operator_channel=$(oc get packagemanifest \
+                -l catalog=${operator_source} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)